src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -339,13 +338,12 @@ export async function getQuickFixesForCommand(
 									run: () => {
 										onDidRequestRerunCommand?.fire({
 											command: fix.terminalCommand,
-											addNewLine: fix.addNewLine
+											addNewLine: !!fix.addNewLine ? fix.addNewLine : true
```

Comment:
I think this will always use true, verify via the free port quick fix. This would be nicer also:

```suggestion
											addNewLine: fix.addNewLine ?? true
```